maps/biter_battles_v2/ai.lua

```diff
@@ -174,12 +174,16 @@ local function spawn_biters(isItnormalBiters, maxLoopIteration,spawner,biter_thr
 		end
 		
 		--Announce New Spawn
+		local team_name = force_name
+		if global.tm_custom_name[force_name] ~= nil then
+			team_name = global.tm_custom_name[force_name]
+		end
 		if(isItnormalBiters and global.biter_spawn_unseen[force_name][unit_name]) then
-			game.print("A " .. unit_name:gsub("-", " ") .. " was spotted far away on team " .. force_name .. "...")
+			game.print("A " .. unit_name:gsub("-", " ") .. " was spotted far away on team " .. team_name .. "...")
```

Comment:
```suggestion
			game.print("A " .. unit_name:gsub("-", " ") .. " was spotted far away on team " .. (global.tm_custom_name[force_name] or force_name) .. "...")
```
Or even better with table:
`game.print({"", "A ", unit_name:gsub("-", " "), " was spotted far away on team ", (global.tm_custom_name[force_name] or force_name), "..."})`